src/main/java/org/broadinstitute/hellbender/utils/diffengine/DiffValue.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.hellbender.utils.diffengine;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * An interface that must be implemented to allow us to calculate differences
+ * between structured objects
```

Comment:
Same inapplicable comment in this class (`DiffValue` is not an interface). Add a proper description for this class.